indico/web/client/styles/modules/event_display/_meetings.scss

```diff
@@ -65,6 +65,14 @@ li > table + .note-area-wrapper {
       color: $black;
     }
 
+    li p {
+      margin-bottom: 0;
+    }
+
+    ul {
+      margin: 0.8em 0;
```

Comment:
`0 0 0.8em 0`?

Reply:
Sounds good, but what do you think about applying this style globally rather than per-component? This seems like something we would like to keep consistent everywhere, and it might end up being easier to keep it global.

Reply:
I would not apply global CSS changes of this kind in a patch release :) Way too likely that it affects some place where we don't want this...